mutiny-core/src/lspclient.rs

```diff
@@ -50,8 +50,20 @@ pub struct ProposalResponse {
     pub jit_bolt11: String,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct FeeRequest {
+    pub pubkey: String,
+    pub amount_msat: u64,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct FeeResponse {
+    pub fee_amount_msat: u64,
+}
+
```

Comment:
for these would be good to derive debug and clone too

Reply:
If and when needed :)